kod/object/active/holder/nomoveon/battler/monster/towns/Goad.kod

```diff
@@ -556,13 +558,6 @@ messages:
 
        if plCombatants = $  { return; }
 
-       if piCombat_style = STYLE_GUILD_VS_GUILD
```

Comment:
Guild combat was never intended for Goad and has none of the related functions.  This looks like a copy and paste problem.